actors/builtin/power/power_actor.go

```diff
@@ -22,6 +22,10 @@ type Runtime = vmr.Runtime
 
 type SectorTermination int64
 
+const (
+	ErrTooManyProveCommits = exitcode.ErrPlaceholder + iota + 1
```

Comment:
`ErrPlaceholder` has to go (#267). Please just hardcode a number here. I suggest `32+iota`. We could add a `FirstActorSpecificExitCode=32` if you like. (I just picked that number then based on the shared codes we have now)